release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1221,7 +1221,7 @@ For pre-IceLake processors, the `intel_pstate` is deactivated by default, wherea
 [id="ocp-4-16-support-for-compute-nodes-on-amd-cpus_{context}"]
 ==== Support for compute nodes with AMD EPYC Zen 4 CPUs
 
-From release 4.16.30, you can use PerformanceProfiles to configure compute nodes on machines equipped with AMD EPYC Zen 4 CPUs such as Genoa and Bergamo. Per-pod power management is currently not supported on AMD.
+From release 4.16.30, you can use PerformanceProfiles to configure compute nodes on machines equipped with AMD EPYC Zen 4 CPUs, such as Genoa and Bergamo. Only single NUMA domain (NPS=1) configurations are supported. Per-pod power management is currently not supported on AMD.
```

Comment:
Same comment as https://github.com/openshift/openshift-docs/pull/89756/files#r1988889199 (back ticks and add object name)